### ISSUE TITLE
Task not found in TaskRegistry

### ISSUE THREAD

**Issue by zN3utr4l** [coleifer/huey#765]
If I create a task in tasks.py, let it run for a while, then comment it out and create another task, the following error is shown:
```

(huey-example) (huey-example) PS C:\inetpub\wwwroot\huey_example> python manage.py run_huey
[2023-09-13 14:15:26,475] INFO:huey.consumer:MainThread:Huey consumer started with 4 thread, PID 14540 at 2023-09-13 12:15:26.475666
[2023-09-13 14:15:26,475] INFO:huey.consumer:MainThread:Scheduler runs every 1 second(s).
[2023-09-13 14:15:26,475] INFO:huey.consumer:MainThread:Periodic tasks are enabled.
[2023-09-13 14:15:26,475] INFO:huey.consumer:MainThread:The following commands are available:
+ test_app.tasks.test_lock
[2023-09-13 14:15:26,480] INFO:huey.consumer.Scheduler:Scheduler:Enqueueing periodic task test_app.tasks.test_lock: 0843a5ab-ce67-49f3-918e-642b832945cc.
[2023-09-13 14:15:33,407] ERROR:huey.consumer.Worker:Worker-3:Error reading from queue
Traceback (most recent call last):
  File "D:\Users\g.chirico\micromamba\envs\huey-example\lib\site-packages\huey\consumer.py", line 109, in loop
    task = self.huey.dequeue()
  File "D:\Users\g.chirico\micromamba\envs\huey-example\lib\site-packages\huey\api.py", line 318, in dequeue
    return self.deserialize_task(data)
  File "D:\Users\g.chirico\micromamba\envs\huey-example\lib\site-packages\huey\api.py", line 290, in deserialize_task
    return self._registry.create_task(message)
  File "D:\Users\g.chirico\micromamba\envs\huey-example\lib\site-packages\huey\registry.py", line 90, in create_task
    TaskClass = self.string_to_task(message.name)
  File "D:\Users\g.chirico\micromamba\envs\huey-example\lib\site-packages\huey\registry.py", line 49, in string_to_task
    raise HueyException('%s not found in TaskRegistry' % task_str)
huey.exceptions.HueyException: test_app.tasks.create not found in TaskRegistry
```

Is there a procedure to clean the TaskRegistry of tasks no longer declared in tasks.py? or make sure that when they are taken (I don't know why they are taken) they are ignored?

**Comment by coleifer** [comment]
This is expected behavior - if you enqueue a task that the consumer does not know how to run, then an error will occur.

**Comment by coleifer** [comment]
You can flush the queue and schedule if you wish to do that, no need to delete the db. https://huey.readthedocs.io/en/latest/api.html#BaseStorage.flush_queue

**Comment by zN3utr4l** [comment]
After adding this to the top of the tasks.py file:

```
from huey.contrib.djhuey import HUEY
HUEY.storage.flush_schedule()
```

This Warning is shown:
```
(huey-example) (huey-example) PS C:\inetpub\wwwroot\huey_example> python manage.py run_huey
[2023-09-13 15:04:11,661] INFO:huey.consumer:MainThread:Huey consumer started with 4 thread, PID 29724 at 2023-09-13 13:04:11.661755
[2023-09-13 15:04:11,677] INFO:huey.consumer:MainThread:Scheduler runs every 1 second(s).
[2023-09-13 15:04:11,677] INFO:huey.consumer:MainThread:Periodic tasks are enabled.
[2023-09-13 15:04:11,677] INFO:huey.consumer:MainThread:The following commands are available:
+ test_app.tasks.test_lock
[2023-09-13 15:04:11,677] INFO:huey.consumer.Scheduler:Scheduler:Enqueueing periodic task test_app.tasks.test_lock: 95a399c8-e1b6-4f41-a481-a37830492573.
[2023-09-13 15:04:11,741] INFO:huey:Worker-2:Executing test_app.tasks.test_lock: 95a399c8-e1b6-4f41-a481-a37830492573
[2023-09-13 15:04:11,763] WARNING:huey:Worker-2:Task 95a399c8-e1b6-4f41-a481-a37830492573 not run, unable to acquire lock.
[2023-09-13 15:05:11,677] INFO:huey.consumer.Scheduler:Scheduler:Enqueueing periodic task test_app.tasks.test_lock: f484881a-2043-42c8-a421-f5545381c85c.
[2023-09-13 15:05:11,699] INFO:huey:Worker-3:Executing test_app.tasks.test_lock: f484881a-2043-42c8-a421-f5545381c85c
[2023-09-13 15:05:11,721] WARNING:huey:Worker-3:Task f484881a-2043-42c8-a421-f5545381c85c not run, unable to acquire lock.
[2023-09-13 15:06:11,655] INFO:huey.consumer.Scheduler:Scheduler:Enqueueing periodic task test_app.tasks.test_lock: 20d24bfa-85ed-4865-b5df-bb2ad50073f6.
[2023-09-13 15:06:11,671] INFO:huey:Worker-3:Executing test_app.tasks.test_lock: 20d24bfa-85ed-4865-b5df-bb2ad50073f6
[2023-09-13 15:06:11,686] WARNING:huey:Worker-3:Task 20d24bfa-85ed-4865-b5df-bb2ad50073f6 not run, unable to acquire lock.
```

**tasks.py**
```
@db_periodic_task(crontab())
@lock_task('test_lock-lock')  # Goes *after* the task decorator.
def test_lock():
    print('Executing test_lock ...')
    sleep(65)
    print('Finish test_lock ...')
```

**Comment by coleifer** [comment]
Yes you may need to flush the lock before starting. You can run huey with `-f` argument.
